### DISCUSSION TITLE
Saving model to JIT, ONNX

### DISCUSSION THREAD

**Discussion by kemjensak** [Toni-SM/skrl#204] [Q&A]
HI, I'm using SKRL with Isaac lab
Is there any method to export model with JIT, ONNX format?
RSL-RL has policy exporter as below:
[jit](https://github.com/IROL-SSU/orbit/blob/416348c073b0f2eb3a311cb18388599d1a54a903/source/extensions/omni.isaac.lab_tasks/omni/isaac/lab_tasks/utils/wrappers/rsl_rl/exporter.py#L11), [onnx](https://github.com/IROL-SSU/orbit/blob/416348c073b0f2eb3a311cb18388599d1a54a903/source/extensions/omni.isaac.lab_tasks/omni/isaac/lab_tasks/utils/wrappers/rsl_rl/exporter.py#L24)

**Comment by Toni-SM** [comment]
Hi @kemjensak 

Sorry for late response.
Currently no, but it is in the skrl [roadmap](https://github.com/users/Toni-SM/projects/2) for a while now, I just haven't had time for it :see_no_evil:.
Contributions are welcome! :sweat_smile:

**Reply by renanmb** [reply]
I am working on it but I am currently stuck how to make a normalizer like RSL_RL, also the constructors are confusing. The code evolved a lot faster than the docs. https://github.com/leggedrobotics/rsl_rl/blob/main/rsl_rl/modules/normalizer.py#L14

**Comment by LemonFoxmere** [comment]
Has there been any updates to this?

**Reply by Toni-SM** [reply]
Hi @LemonFoxmere 

There is a PR in progress that implements this feature: https://github.com/Toni-SM/skrl/pull/410
